library/src/rocblas_auxiliary.cpp

```diff
@@ -59,6 +59,28 @@ extern "C" rocblas_status rocblas_set_pointer_mode(rocblas_handle handle, rocbla
     return rocblas_status_success;
 }
 
+#ifdef USE_TENSILE_HOST
+extern "C" rocblas_status rocblas_create_host_handle(rocblas_handle* handle, const char* lib_path)
+{
```

Comment:
We should not add  `rocblas_create_host_handle` -- `rocblas_create_handle` should work with the new or old Tensile code. If `lib_path` is needed for a library path, then it needs to be in an environment variable, such as `ROCBLAS_TENSILE_LIBPATH` or something else.

The normal `rocblas_create_handle` API and mechanism should initialize Tensile Host without requiring the user to call `rocblas_create_host_handle`.

Reply:
Being addressed in refactoring.

Reply:
Is there an example of how to handle environment variables in rocblas?

Reply:
> 
> 
> Is there an example of how to handle environment variables in rocblas?

See my PR: https://github.com/wbgilmartin/rocBLAS/pull/1